tests/testcases.py

```diff
@@ -208,3 +217,9 @@ def assertImageDoesNotExist(self, image):
 
     def assertExecDocker(self, *args):
         self.fake_exec_docker.assert_called_with(*args)
+
+    def inspect_container(self, service_name):
+        client = get_docker_client()
+        container_id = _get_container_for_app_or_service(client, service_name, include_exited=True)['Id']
+        return client.inspect_container(container_id)
```

Comment:
Can you put this definition above the asserts for the sake of keeping some sort of structure? It's a big hairy class already.